journals/apps/journals/models.py

```diff
@@ -320,6 +322,51 @@ def generate_basket_url(self, sku):
         ))
         return basket_url
 
+
+
+    def update_related_objects(self, deactivate=False):
+
+        def update_service(client, data, service_name):
+            try:
+                client.journals(self.journal.uuid).patch(data)
+            except HttpNotFoundError as err:
+                # Only a WARN because this will often happen on JournalAboutPage creation.
+                logging.warn(f"JournalAboutPage unable to update {service_name} because UUID doesn't exist: {err.content}")
+            except HttpClientError as err:
+                logging.error(f"Error updating discovery after JournalAboutPage publish: {err.content}")
+
+        discovery_data = {
+            "status": "active" if not deactivate else "inactive",
+            "card_image_url": self.card_image_absolute_url,
+            "title": self.title,
+            "full_description": self.long_description,
+            "short_description": self.short_description,
+        }
+        update_service(
+            self.site.siteconfiguration.discovery_journal_api_client,
+            discovery_data,
+            "discovery"
+        )
+
+        ecommerce_data = {
+            "title": self.title
+        }
+        update_service(
```

Comment:
@tuchfarber 
Does this post fail because we only supply title? If so, remove this for now and please create a story for Beta2 to expose ecommerce API that allows changing only title.